script.js

```diff
@@ -1,12 +1,46 @@
 //You can edit ALL of the code here
+const getBody = document.querySelector("body");
+let makeDiv = document.createElement("div");
+makeDiv.id = "main";
+getBody.appendChild(makeDiv);
+const allEpisodes = getAllEpisodes();
+
 function setup() {
   const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
 }
 
+function pad2(number) {
+  return (number < 10 ? "0" : "") + number;
+}
+
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  // const rootElem = document.getElementById("root");
+  // rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  for (let i = 0; i < episodeList.length; i++) {
+    const makeCard = document.createElement("div");
```

Comment:
This function is a little bit too long at the moment - you could abstract some of the logic into another that creates a single episode card